<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="rut_bar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="codigo_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rut</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre_bar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nombre_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cargo_bar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cargo_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="genero_bar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="genero_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Genero</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sueldo_bar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sueldo_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sueldo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Local</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="local_bar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
